Failure on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.35548&attempt_id=73b0d8d4-b743-49f9-b872-ef2349db8a6b)

<details><summary>Partial log (click to expand)</summary><p>

```
make[3]: *** [../../config/Makefile:24: versdep.cmo] Error 127
make[3]: Leaving directory '/build/source/ocaml_src/lib'
make[2]: *** [Makefile:164: library_cold] Error 2
make[2]: Leaving directory '/build/source'
make[1]: *** [Makefile:35: boot/camlp5] Error 2
make[1]: Leaving directory '/build/source'
make: *** [Makefile:150: world.opt] Error 2
builder for '/nix/store/y57dzhha3nhk7rs3bvdacsmyfz4cr5sa-camlp5_transitional-7.05.drv' failed with exit code 2
builder for '/nix/store/c7i80l4r67b2ybkhws5lvbd14dzamk7d-camlp5-7.05.drv' failed with exit code 2
[31;1merror:[0m build of '/nix/store/c7i80l4r67b2ybkhws5lvbd14dzamk7d-camlp5-7.05.drv', '/nix/store/y57dzhha3nhk7rs3bvdacsmyfz4cr5sa-camlp5_transitional-7.05.drv' failed
```
</p></details>

--- a/pkgs/development/tools/ocaml/camlp5/default.nix
+++ b/pkgs/development/tools/ocaml/camlp5/default.nix
@@ -6,14 +6,21 @@ in
 
 stdenv.mkDerivation {
 
-  name = "camlp5${if transitional then "_transitional" else ""}-7.03";
+  name = "camlp5${if transitional then "_transitional" else ""}-7.05";
 
   src = fetchzip {
-    url = https://github.com/camlp5/camlp5/archive/rel703.tar.gz;
-    sha256 = "0bwzhp4qjypfa0x8drp8w434dfixm1nzrm6pcy9s5akpmqvb50a8";
+    url = https://github.com/camlp5/camlp5/archive/rel705.tar.gz;
+    sha256 = "16igfyjl2jja4f1mibjfzk0c2jr09nxsz6lb63x1jkccmy6430q2";
   };
 
   buildInputs = [ ocaml ];
+
+  postPatch = ''
+    for p in compile/compile.sh config/Makefile.tpl test/Makefile test/check_ocaml_versions.sh
+    do
+      substituteInPlace $p --replace '/bin/rm' rm
+    done
+  '';
 
   prefixKey = "-prefix ";
 

